h/accounts/views.py

```diff
@@ -465,116 +404,111 @@ def _register(self, username, email, password):
         self.request.registry.notify(RegistrationEvent(self.request, user))
 
 
+@view_defaults(route_name='profile',
+               renderer='h:templates/accounts/profile.html.jinja2')
+@view_config(attr='profile_form', request_method='GET')
+@view_config(attr='profile', request_method='POST')
```

Comment:
I've never understood this pattern in h of putting `@view_config`s on the class. It just means they're far away from the method they refer to. The way the docs show to do this is with `@view_defaults` on the class but `@view_config` on the methods: <http://docs.pylonsproject.org/projects/pyramid/en/1.5-branch/narr/viewconfig.html#view-defaults-class-decorator>. I guess there's an argument to be made for following the existing style in h though.